lib/minitest/ci.rb

```diff
@@ -48,14 +48,16 @@ def record result
       results[result.class] << result
     end
 
+    ##
+    # Generate test report
     def report
       io.puts
-      io.puts 'generating ci files'
+      io.puts '[Minitest::CI] Generating test report in JUnit XML format...'
 
       Dir.chdir report_dir do
         results.each do |name, resultz|
```

Comment:
Should this be `results.each do |name, result|`? Not sure why the block param is named `resultz`...